packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -433,7 +433,7 @@ class Frame extends React.Component {
       || event.target.hasAttribute('data-terra-form-select-toggle-button-icon'))) {
       this.setState({ isOpen: true, isPositioned: false });
 
-      FrameUtil.shiftFocusToMenu(this);
+      setTimeout(() => FrameUtil.shiftFocusToMenu(this), 100);
```

Comment:
FrameUtil.shiftFocusToMenu() has already has wait time of 100ms. this will add up 100ms more wait time to that function. if increasing time is the only way to fix this then we can do it in FrameUtil itself instead adding it in each frame files.

Have you tried using useLayoutEffect() hook to identify whether the menu component is loaded up with items and then shifting focus to menu..?